src/chapters/configuring.xml

```diff
@@ -1687,7 +1693,7 @@ tcp6       0      0 :::4713                 :::*                    LISTEN</comp
                                         <varlistentry>
                                             <term><constant>fr-ch-qwertz</constant></term>
                                             <listitem>
-                                                <para>Swiss French keyboard (qwertz)</para>
+                                                <para>French (Swiss) keyboard (qwertz)</para>
```

Comment:
I believe the correct name is as originally written - "Swiss French":

https://en.wikipedia.org/wiki/Swiss_French